### PR TITLE
📄 Add Brainstorming Services page

### LINKED ISSUE #1719: 📄  Brainstorming page
### 1. People in the loop:

Just me - I wanted to make this page

### 2. Page template to copy:

https://www.ssw.com.au/consulting/chatbots

### 3. CRM Record (only for consulting service pages):

https://ssw.crm6.dynamics.com/main.aspx?appid=4b3512af-9675-eb11-a812-002248151e0c&pagetype=entityrecord&etn=ssw_consultingservice&id=5019a147-2084-ee11-8179-6045bd3df63e

### 4. URI:

consulting/brainstorming

### 5. Page Title:

 Brainstorming Services for Innovative Solutions

### 6. Meta description:

Transform your team's untapped ideas into business solutions with SSW's Brainstorming Days. an engaging, collaborative environment.

### 8. Content:

Embracing innovation and employee empowerment, SSW has significantly benefited from hosting Brainstorming Days. These engaging and enjoyable sessions have sparked a wave of creative solutions, leading to the introduction and enhancement of many internal processes.

By combining fun with functional, we've created a platform where our employees' ideas are not only heard but also transformed into actionable business improvements. Leveraging tools like Microsoft Azure DevOps and GitHub, these brainstorming sessions exemplify our commitment to collaborative innovation and continuous improvement.

CULTURE OF INNOVATION
Regular brainstorming cultivates a culture of innovation and continuous improvement, positioning the company as a forward-thinking leader in its field.

DIVERSE PERSPECTIVES, BETTER SOLUTIONS
Brainstorming Days bring together diverse employee perspectives, fostering an environment where unique, innovative solutions are born. This diversity in thought leads to breakthroughs in problem-solving and creative thinking.

IDENTIFICATION OF EFFICIENCIES
By tapping into the collective intelligence of the team, Brainstorming Days often lead to the identification of new efficiencies and process improvements, directly impacting the bottom line.

FUN AND ENGAGING
Beyond business benefits, these sessions are fun and engaging, boosting employee satisfaction and fostering a positive workplace environment.

### PR DESCRIPTION
**Pull request by bradystroud**
Affected routes: 
- `/consulting/brainstorming`

#1719

- [x] If adding a new page, I have followed the [SEO checklist](https://www.ssw.com.au/rules/seo-checklist/)

image

**Figure: Brainstorming page**

### CODE REVIEW

File: content/consulting/index/index.json [hunk 1]
```diff
@@ -241,7 +241,7 @@
         {
           "title": "Power Automate Process Mining",
           "description": "Process mining uncovers processes in your company that can be effectively automated with artificial intelligence, so you save costs and increase your productivity.",
-          "logo": "/images/thumbs/thumb-power-automate-icon.jpg",
+          "logo": "",
```
Comment (GordonBeeming) [inline content/consulting/index/index.json]: Is this logo being removed intended as part of this pr?

Comment (bradystroud) [reply content/consulting/index/index.json]: Mistake thanks

Comment (bradystroud) [reply content/consulting/index/index.json]: fixed in latest commit

**Review by GordonBeeming — APPROVED** [review]

**Comment by camillars** [comment]
@bradystroud 

@UlyssesMaclaren and I were not tagged as per the standard...

1. is this a new page?
2. is there an email about this?

**Comment by bradystroud** [comment]
Sorry @camillars @UlyssesMaclaren 

I did not follow a standard.

> 1. is this a new page?

Yes

> 2. is there an email about this?

No
I created this page because I wanted it for CTF this week

**Comment by sethdaily** [comment]
(checked by Brady)
@bradystroud 

1. Please follow the [New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) template so you have done the process correctly and there is a record of the source

**Comment by bradystroud** [comment]
> 1. Please follow the [New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) template so you have done the process correctly and there is a record of the source

Done - https://github.com/SSWConsulting/SSW.Website/issues/1719

**Pull request merged**